test/unittests/CandidateTest.php

```diff
@@ -244,6 +271,271 @@ public function testSetDataWithValueSucceeds()
+     *
+     * @covers Candidate::getCandidateSite
+     * @return void
+     */
+    public function testGetCandidateSite()
+    {
+        $this->_setUpTestDoublesForSelectCandidate();
+        $this->_candidate->select($this->_candidateInfo['CandID']);
+
+        $this->assertEquals(
+            $this->_candidateInfo['PSC'],
+            $this->_candidate->getCandidateSite()
+        );
+    }
+
+    /**
+     * Test getCenterID returns the correct RegistrationCenterID for the candidate
+     *
+     * @covers Candidate::getCenterID
+     * @return void
+     */
+    public function testGetCenterID()
+    {
+        $this->_setUpTestDoublesForSelectCandidate();
+        $this->_candidate->select($this->_candidateInfo['CandID']);
+
+        $this->assertEquals(
+            $this->_candidateInfo['RegistrationCenterID'],
+            $this->_candidate->getCenterID()
+        );
+    }
+
+    /**
+     * Test getCandidateDoB returns the correct DoB for the candidate
+     *
+     * @covers Candidate::getCandidateDoB
+     * @return void
+     */
+    public function testGetCandidateDoB()
+    {
+        $this->_setUpTestDoublesForSelectCandidate();
+        $this->_candidate->select($this->_candidateInfo['CandID']);
+
+        $this->assertEquals(
+            $this->_candidateInfo['DoB'],
+            $this->_candidate->getCandidateDoB()
+        );
+    }
+
+    /**
+     * Test getCandidateEDC returns the correct 
+     * expected date of confinement for the candidate
+     *
+     * @covers Candidate::getCandidateEDC
+     * @return void
+     */
+    public function testGetCandidateEDC()
+    {
+        $this->_setUpTestDoublesForSelectCandidate();
+        $this->_candidate->select($this->_candidateInfo['CandID']);
+
+        $this->assertEquals(
+            $this->_candidateInfo['EDC'],
+            $this->_candidate->getCandidateEDC()
+        );
+    }
+
+    /**
+     * Test getCandidateSex returns the correct sex of the candidate
+     *
+     * @covers Candidate::getCandidateSex
+     * @return void
+     */
+    public function testGetCandidateSex()
+    {
+        $this->_setUpTestDoublesForSelectCandidate();
+        $this->_candidate->select($this->_candidateInfo['CandID']);
+
+        $this->assertEquals(
+            $this->_candidateInfo['Sex'],
+            $this->_candidate->getCandidateSex()
+        );
+    }
+
+    /**
+     * Test getCandidateEthnicity returns the correct ethnicity of the candidate
+     *
+     * @note   This function is deprecated so it was causing a PHAN error. 
+     *         It is marked test incomplete for this reason
+     *         however it is fully implemented and functional
+     * @covers Candidate::getCandidateEthnicity
+     * @return void
+     */
+    public function testGetCandidateEthnicity()
+    {
+        $this->markTestIncomplete(
+            "This test is marked as incomplete because the function is deprecated"
+        );
```

Comment:
I think you should just delete it, then (also GitHub is highlighting the commented out code as an error because of the `/**`